changelogs/unreleased/80473.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "PeriodInput: add defaultInterval to set a default interval between start and end dates",
```

Comment:
"PeriodInput: add interval in between start and end dates with possibility to define a default one in hours with props"